Entity/CustomObject.php

```diff
@@ -462,4 +472,20 @@ public function recordCustomFieldChanges()
             $this->addChange("customfield:{$deletedField['id']}", [null, 'deleted']);
         }
     }
+
+    /**
+     * @param ArrayCollection $uniqueIdentifierFields
+     * @param array<mixed> $rowData
+     * @return string|null
+     */
```

Comment:
```suggestion
    /**
     * @param ArrayCollection<int,string> $uniqueIdentifierFields
     * @param array<mixed> $rowData
     */
```

Reply:
Thanks for the suggestions @escopecz. Committed these changes.